crates/rooch/src/commands/init.rs

```diff
@@ -23,6 +24,9 @@ pub struct Init {
     /// Command line input of custom server URL
     #[clap(short = 's', long = "server-url")]
     pub server_url: Option<String>,
+    /// Whether a password should be provided.
+    #[clap(short = 'p', long = "password")]
+    password_required: bool,
```

Comment:
Make `password_required` default value to `true`? And we can set it to `false` in the test case.

So, if we want to let `password_required` accept value, it should be `Option<bool>`.


```
#[clap(short = 'p', long = "password")]
password_required: Option<bool>,
```

Reply:
I've added it.